src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -537,6 +557,7 @@ void LogError(IRestResponse response)
         static bool HasSucceeded(IRestResponse response) => successCodes.Any(x => response != null && x == response.StatusCode && response.ErrorException == null);
 
         static IEnumerable<HttpStatusCode> successCodes = new[] { HttpStatusCode.OK, HttpStatusCode.Accepted, HttpStatusCode.NotModified, HttpStatusCode.NoContent };
+        static Regex linkExpression = new Regex(@"<([^>]+)>;\s?rel=""(\w+)"",?\s?", RegexOptions.Compiled);
```

Comment:
In your [previous comment](https://github.com/Particular/ServiceInsight/pull/737/files#r127168145), I see no mention of `<` and `>`. Could you please supply an example of the actual header value so I can understand better?